src/testUtils.tsx

```diff
@@ -13,9 +14,9 @@ export const client = new ApolloClient({
 
 function Providers({children}: {children: React.ReactNode}) {
   return (
-    <ThemeProvider>
-      <ApolloProvider client={client}>{children}</ApolloProvider>
-    </ThemeProvider>
+    <LitentryApiClientProvider>
```

Comment:
Can you please create a ticket to clean this up later. In the LitentryApiClientProvider we have an instance of Apollo client already. And we are creating a new Apollo client here also, which is needed in the `setupTests.js` file. Ideally we would use the same instance of the Apollo client in test environment too.

For now this implementation unblocks our tests, but please include the ticket here with a TODO.

Reply:
created an issue here: https://github.com/litentry/litentry-app/issues/1275